package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "default-branch",
-  "version": "1.0.8",
+  "version": "1.0.9",
```

Comment:
Can you revert the version upgrade? I would like to put it in a separate commit and probably bump the version further.

Reply:
Sure, no problem. Already done :)

Reply:
Thanks! 😄